beta-src/.gitignore

```diff
@@ -11,6 +11,4 @@
 .env.local
 .env.development.local
 .env.test.local
-.env.production.local
-
-npm-debug.log*
```

Comment:
Why are we removing `npm-debug.log`?

Reply:
Whoops! That might have been removed by accident. I'll add it back lol.